src/components/TheFloatingButton.vue

```diff
@@ -0,0 +1,71 @@
+<template>
+  <button class="wrapper" @click="toggleAppModal">
+    <i class="fa-solid fa-plus"></i>
+  </button>
+</template>
+
+<script>
+import { mapMutations } from 'vuex';
+import { TOGGLE_APP_MODAL } from '@/store/mutation-types';
```

Comment:
This button should not be able to close the modal from a UX perspective
it should just open the modal, maybe you could use a boolean for toggling